tests/common/snappi/snappi_fixtures.py

```diff
@@ -517,3 +522,371 @@ def cvg_api(snappi_api_serv_ip,
+
+        """ For each physical interface attached to this Vlan """
+        for i in range(len(phy_intfs)):
+            phy_intf = phy_intfs[i]
+            vlan_ip_addr = vlan_ip_addrs[i]
+
+            port_ids = [id for id, snappi_port in enumerate(snappi_ports)
+                        if snappi_port['peer_port'] == phy_intf]
+            if len(port_ids) != 1:
+                return False
+
+            port_id = port_ids[0]
+            mac = __gen_mac(port_id)
+            device = config.devices.device(
+                name='Device Port {}'.format(port_id))[-1]
+
+            ethernet = device.ethernets.add()
+            ethernet.name = 'Ethernet Port {}'.format(port_id)
+            ethernet.port_name = config.ports[port_id].name
+            ethernet.mac = mac
+
+            ip_stack = ethernet.ipv4_addresses.add()
+            ip_stack.name = 'Ipv4 Port {}'.format(port_id)
+            ip_stack.address = vlan_ip_addr
+            ip_stack.prefix = int(prefix)
+            ip_stack.gateway = gw_addr
+
+            port_config = SnappiPortConfig(id=port_id,
+                                           ip=vlan_ip_addr,
+                                           mac=mac,
+                                           gw=gw_addr,
+                                           gw_mac=dut_mac,
+                                           prefix_len=prefix,
+                                           port_type=SnappiPortType.VlanMember,
+                                           peer_port=phy_intf)
+
+            port_config_list.append(port_config)
+
+    return True
+
+
+def __intf_config_multidut(config, port_config_list, duthost, snappi_ports):
+    """
+    Configures interfaces of the DUT
+    Args:
+        config (obj): Snappi API config of the testbed
+        port_config_list (list): list of Snappi port configuration information
+        duthost (object): device under test
+        snappi_ports (list): list of Snappi port information
+    Returns:
+        True if we successfully configure the interfaces or False
+    """
+    dutIps = create_ip_list(dut_ip_start, len(snappi_ports), mask=prefix_length)
+    tgenIps = create_ip_list(snappi_ip_start, len(snappi_ports), mask=prefix_length)
+    ports = [port for port in snappi_ports if port['peer_device'] == duthost.hostname]
+
+    for port in ports:
+        port_id = port['port_id']
+        dutIp = dutIps[port_id]
+        tgenIp = tgenIps[port_id]
+        mac = __gen_mac(port_id)
+        logger.info('Configuring Dut: {} with port {} with IP {}/{}'.format(
+                                                                            duthost.hostname,
+                                                                            port['peer_port'],
+                                                                            dutIp,
+                                                                            prefix_length))
+        if port['asic_value'] == 'None':
```

Comment:
if port['asic_value'] == 'None' -- wrong
It must be without quotes
if port['asic_value'] == None -- correct

Reply:
this has been resolved